azkaban-common/src/main/java/azkaban/constants/ServerProperties.java

```diff
@@ -55,4 +55,8 @@
 
   // The property is used for the web server to get the host name of the executor when running in SOLO mode.
   public static final String EXECUTOR_HOST = "executor.host";
+
+  // Max flow running time in mins, server will kill flows running longer than this setting.
+  // if not set or <= 0, then there's no restriction on running time.
+  public static final String AZKABAN_MAX_FLOW_RUNNING_MINS = "azkaban.server.flow.max.running.minutes";
```

Comment:
So @chengren311, what does it mean that "if not set .. there's no restriction on running time"? Isn't that comment wrong then?